contracts/SnappAuction.sol

```diff
@@ -56,37 +78,90 @@ contract SnappAuction is SnappBase {
         return auctions[slot].appliedAccountStateIndex != 0;
     }
 
+    function getStandingOrderHash(uint16 userId, uint128 batchIndex) public view returns (bytes32) {
+        return standingOrders[userId].reservedAccountOrders[batchIndex].orderHash;
+    }
+    
+    function getStandingOrderValidFrom(uint16 userId, uint128 batchIndex) public view returns (uint) {
+        return standingOrders[userId].reservedAccountOrders[batchIndex].validFromIndex;
+    }
+
+    function getStandingOrderValidTo(uint16 userId, uint128 batchIndex) public view returns (uint) {
+        uint validTo = standingOrders[userId].reservedAccountOrders[batchIndex + 1].validFromIndex;
+        if (validTo == 0) {
+            return MAX_UINT;
+        } else {
+            return validTo - 1; 
+        }
+    }
+
+    function getStandingOrderCounter(uint16 userId) public view returns (uint) {
+        return standingOrders[userId].currentBatchIndex;
+    }
+
     /**
      * Auction Functionality
      */
+    function placeStandingSellOrder(
+        uint8[] memory buyTokens,
+        uint8[] memory sellTokens,
+        uint128[] memory buyAmounts,
+        uint128[] memory sellAmounts
+    ) public onlyRegistered() {
+        
+        // Update Auction Hash based on request
+        uint16 accountId = publicKeyToAccountMap(msg.sender);
+        require(accountId <= AUCTION_RESERVED_ACCOUNTS, "Accout is not a reserved account");
+
+        bytes32 orderHash;
+        uint numOrders = buyTokens.length;
+        require(numOrders <= AUCTION_RESERVED_ACCOUNT_BATCH_SIZE, "Too many orders for reserved batch");
+        
+        if (
+            auctionIndex == MAX_UINT ||
+            block.timestamp > (auctions[auctionIndex].creationTimestamp + 3 minutes)
+        ) {
+            createNewPendingBatch();
```

Comment:
Why are we doing this here? These batches are not affected by the code that follows. I see no reason to update the unreserved order batch.

Reply:
answered in the issue